repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -875,6 +930,60 @@ export const getPeriodAvailabilityElement = (
     },
 });
 
+export const getExemptionsElement = (
+    id: string,
+    validityParametersObject: {},
+    hasTimeRestriction: boolean,
+    productName?: string,
+    groupOfLinesRef?: string,
+): NetexObject => ({
+    version: '1.0',
+    id: `op:${id}`,
+    Name: { $t: 'Exempted Services' },
+    TypeOfFareStructureElementRef: {
+        version: 'fxc:v1.0',
+        ref: hasTimeRestriction ? 'fxc:access_when' : 'fxc:access',
+    },
+    qualityStructureFactors: hasTimeRestriction
+        ? {
+              FareDemandFactorRef: {
+                  ref: 'op@Tariff@Demand',
+                  version: '1.0',
+              },
+          }
+        : null,
+    GenericParameterAssignment: {
+        id,
+        version: '1.0',
+        order: '1',
+        TypeOfAccessRightAssignmentRef: {
+            version: 'fxc:v1.0',
+            ref: 'fxc:cannot_access',
+        },
+        ValidityParameterGroupingType: { $t: 'NOT' },
+        validityParameters: validityParametersObject,
+        includes: groupOfLinesRef
```

Comment:
I dont see a scenario where we dont have a groupsOfLinesRef

Reply:
done in other functions such as getLines list it impacts other netex if I remove it